### PR TITLE
Styling/sandra

### PR DESCRIPTION
**Pull request by Sandz-Studio**
A LOT of tailwind changes/ clean-up.  Not including responsive mobile nav.

EDIT: added some fixes for mobile responsiveness, and fixed the broken footer in dev. not changes to compare as not a code issue when branch was being reviewed.

EDIT 2: Noticed budget buttons were not mobile responsive, fixed

### CODE REVIEW

File: src/components/GiftComparisonCard.jsx
Comment (RobyOneJ) [inline src/components/GiftComparisonCard.jsx:1]: will adjust image size and padding; text wrapping/overflow; card containers width and margin for appropriate spacing and overflow handling on this page especially when only 1 or 2 items are added to compare.

Comment (Sandz-Studio) [reply src/components/GiftComparisonCard.jsx:1]: Issue resolved was not connected to the live back end so not showing true gift data. Not a code problem.

**Comment by lruizper** [comment]
## Avoid this rabbit for demo
image

**Review by lruizper — APPROVED** [review]
Everything lools neat and tidy
Let's avoid the bunnies for demo as per prev comment

**Pull request merged**